app/Http/Controllers/Admin/HomeController.php

```diff
@@ -6,8 +6,9 @@
 
 class HomeController extends Controller
 {
+    const INDEX_NAV = 0;
```

Comment:
cách 1 dòng biến với function ra e nhé
với INDEX_NAV e dùng cho mục đích gì á, a chưa rõ

Reply:
![image](https://github.com/anhlp1212/bog_laravel/assets/147794362/053fa268-6c15-4b97-8794-4cf0c22907e7)
![image](https://github.com/anhlp1212/bog_laravel/assets/147794362/4d31515f-4c92-4335-8769-34300709c109)

Dạ em dùng INDEX_NAV để xác định vị trí thêm classList (để thay cái background màu) khi mà chuyển trang ạ.
Em dùng INDEX_NAV = 0 là của trang Dashboard, còn NDEX_NAV = 1 là Posts

Reply:
à là e đang dùng js để thêm class active đúng không
ở php khi click sẽ load lại page nên mình không cần sử dụng js e nhé, dùng cũng được nhưng không cần thiết, ít ai dùng như vậy, khó maintain
e có thể sử dụng 
```
class="{{ request()->routeIs('route_name') ? 'active' : ''
hoặc
request()->route()->getController() === 'controller_name' ? 'active' : ''
```
e có thể dùng những cách khác, sao cho ở những trường hợp tương tự vậy, mình sẽ xử lý cho nó linh động, hạn chế set cứng, ví dụ ở đây nếu a mong muốn e đem Notifications hoặc thêm những giá trị mới trước Post hay Dashboard thì mình sẽ phải đi sửa lai toàn bộ INDEX_NAV này á. không hiệu quả khi maintain sau này.
p/s e chuyển sang dùng jquery hen. e dùng js thì tốt á nhưng không cần thiết lắm, performance không đáng kể